docker/test/fasttest/run.sh

```diff
@@ -9,7 +9,11 @@ trap 'kill $(jobs -pr) ||:' EXIT
 stage=${stage:-}
 
 # Compiler version, normally set by Dockerfile
-export LLVM_VERSION=${LLVM_VERSION:-17}
+export LLVM_VERSION=${LLVM_VERSION:-18}
+
+# For some reason, it is needed for tests with 'expect', but I don't know, why.
+sudo groupadd --system --gid 1000 clickhouse
+sudo useradd --system --gid 1000 --uid 1000 clickhouse
```

Comment:
```
+ sudo groupadd --system --gid 1000 clickhouse
/run.sh: line 15: sudo: command not found
```

You can add `sudo` here - https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/fasttest/Dockerfile